.github/workflows/pr-realm-js.yml

```
@@ -0,0 +1,334 @@
+    outputs:
+      deployedApps: ${{ steps.deploy-mdb-apps.outputs.deployedApps }}
+    timeout-minutes: 20
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: false
+
+      - uses: realm/ci-actions/mdb-realm/deployApps@800a0234bcbe9073e7a07a66c864d54db3688e56
+        id: deploy-mdb-apps
+        with:
+          projectId: ${{ secrets.ATLAS_PROJECT_ID }}
+          apiKey: ${{ secrets.ATLAS_PUBLIC_API_KEY }}
+          privateApiKey: ${{ secrets.ATLAS_PRIVATE_API_KEY }}
+          differentiator: realm js
+
+  react-native-android:
+    name: React Native on Android (${{ matrix.type }})
+    needs: [build-and-test, baas-realm-js]
+    env:
+      MOCHA_REMOTE_TIMEOUT: 10000
+      REALM_DISABLE_ANALYTICS: 1
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        #TODO: Reactivate debug when builds are optimized
+        #type: [Release, Debug]
+        type: [Release]
+    timeout-minutes: 60
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+
+      - name: Setup node version
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+
+      - name: Setup NPM version
+        run: npm install -g npm@7
+
+      - name: Get NPM cache directory
+        id: npm-cache-dir
+        run: echo "::set-output name=dir::$(npm config get cache)"
+
+      - name: Restore NPM cache
+        id: npm-cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.npm-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      # Setup tools
+      - name: Install tools
+        run: brew install watchman
+
+      # download prebuilds
+      - name: Download prebuilds
+        uses: actions/download-artifact@v2
+        with:
+          name: realm-js-prebuilds
+
+      # Install node dependencies
+      - name: Install prebuilds
+        run: npm ci
+
+      # Bootstrap lerna sub-packages (builds the packages, the Realm JS native module and pod install)
+      - name: Bootstrap lerna sub-packages
+        run: npx lerna bootstrap --scope realm-react-native-tests --include-dependencies
+
+      # Run the tests
+      - name: Run tests (Android / Native)
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
+          # ndk: 21.0.6113669
+          # cmake: 3.18.1
+          target: google_apis
+          script: npm run test:android --prefix integration-tests/environments/react-native
+        env:
+          MOCHA_REMOTE_REPORTER: mocha-github-actions-reporter
+          MOCHA_REMOTE_EXIT_ON_ERROR: true
+          SPAWN_LOGCAT: true
+          REALM_BASE_URL:  ${{ secrets.REALM_BASE_URL }}
```

It looks [from the logs](https://github.com/realm/realm-js/runs/5003509590?check_suite_focus=true#step:11:22) like this secret isn't set. Ideally this would be an output from the `baas-realm-js` job to ensure we're testing against the same deployment that we created a cluster on.

As a workaround it can be manually set to [the same value that the GitHub Action defaults to](https://github.com/realm/ci-actions/blob/main/mdb-realm/src/helpers.ts#L107):

```suggestion
          REALM_BASE_URL: "https://realm-dev.mongodb.com"
```